1.9/usage/managing-services/private-docker-registry.md

```diff
@@ -0,0 +1,77 @@
+---
+post_title: Using a Private Docker Registry 
+menu_order: 004.5
+---
+
+To supply credentials to pull from a private registry, add a `docker.tar.gz` file to
+the `uris` field of your app. The `docker.tar.gz` file should include the `.docker` folder that contains
+`.docker/config.json`
+
+# Step 1: Tar/Gzip credentials
+
+1. Log in to the private registry manually. Login creates a `.docker` folder and a `.docker/config.json` file in your home directory.
+
+    ```bash
+      docker login some.docker.host.com
+      Username: foo
+      Password:
+      Email: foo@bar.com
+    ```
+
+1. Tar this folder and its contents.
+
+    ```bash
+    cd ~
+    tar -czf docker.tar.gz .docker
+    ```
+1. Check you have both files in the tar.
+
+    ```bash
+      tar -tvf ~/docker.tar.gz
+
+      drwx------ root/root         0 2015-07-28 02:54 .docker/
+      -rw------- root/root       114 2015-07-28 01:31 .docker/config.json
+    ```
+
+1. Put the gzipped file in location that you will then reference in your application definition.
```

Comment:
`Put the archive file in a location that is accessible by your application definition.`